doalerts/perfherder_alerts.py

```diff
@@ -119,31 +142,58 @@ def analyzeData(sig, branch, framework):
 
 
 def filterUniqueAlerts(results):
-    filtered = [{'sig': {}, 'result': []},{'sig': {}, 'result': []}]
+    filtered = [{"sig": {}, "result": []}, {"sig": {}, "result": []}]
     # TODO: consider weekends or slow times for a longer window - 24 hours
     # TODO: sanity check no alerts are on the same 12 hour window (maybe within 100 pushids?)
     hours = 6
-    first = [datetime.datetime.fromtimestamp(float(x.push_timestamp)) for x in results[0]['result']]
-    second = [datetime.datetime.fromtimestamp(float(x.push_timestamp)) for x in results[1]['result']]
+    first = [
+        datetime.datetime.fromtimestamp(float(x.push_timestamp))
+        for x in results[0]["result"]
+    ]
+    second = [
+        datetime.datetime.fromtimestamp(float(x.push_timestamp))
+        for x in results[1]["result"]
+    ]
     f_matched = set()
     s_matched = set()
     for s in first:
-        f_matched.update([s for f in second if s < f + datetime.timedelta(hours=hours) and s > f - datetime.timedelta(hours=hours) and s not in list(f_matched)])
+        f_matched.update(
+            [
+                s
+                for f in second
+                if s < f + datetime.timedelta(hours=hours)
+                and s > f - datetime.timedelta(hours=hours)
+                and s not in list(f_matched)
+            ]
+        )
```

Comment:
in this case a regular for loop would be easier to read

Reply:
Do you mean use a for loop to build the list instead of using list comprehension? To be clear the only change here is the introduction of new lines by black.